### PR TITLE
Bib. Item: Set attr. for ext. links

### PR DESCRIPTION
**Pull request by mjansenDatabay**
Mantis issue: 22904

If accepted, this should be merged to all maintained releases.

### CODE REVIEW

File: Services/UIComponent/Button/classes/class.ilLinkButton.php [hunk 1]
```diff
@@ -79,7 +79,27 @@ protected function renderCaption()
 	{
 		return '&nbsp;'.$this->getCaption().'&nbsp;';
 	}
-	
+
```
Comment (chfsx) [inline Services/UIComponent/Button/classes/class.ilLinkButton.php:82]: @alex40724 is this part OK for you?

**Review by chfsx — APPROVED** [review]
LGTM, the part in ilLinkButton should be checked by @alex40724

File: Services/UIComponent/Button/classes/class.ilLinkButton.php (hunk 1 above)
Comment (alex40724) [reply Services/UIComponent/Button/classes/class.ilLinkButton.php:82]: Yes, ok. Thanks.

**Review by alex40724 — APPROVED** [review]
Ok for me. Thanks.

**Pull request merged**